src/DocumentFormat.OpenXml.Framework/OpenXmlElement.cs

```diff
@@ -473,6 +473,208 @@ internal set
 
         #region public methods
 
+        private bool PrefixAndQNameEqual(OpenXmlElement other, OpenXmlElementEqualityOptions options)
```

Comment:
I'd prefer these comparisons to be on the equality comparer itself - if you need any private properties, feel free to expose them as internal